### ISSUE TITLE
conftest pull error

### ISSUE THREAD

**Issue by MikaelSmith** [open-policy-agent/conftest#397]
When I run
```
$ conftest --version
Version: 0.21.0
Commit: 125160d
Date: 2020-09-13T10:21:35Z
$ conftest pull github.com/instrumenta/policies/kubernetes -p policy/kubesec
Error: download policies: client get: RemoveAll .: invalid argument
```
on macOS 10.15.6 I get an error. This used to work.

Installed via Homebrew.

**Comment by jpreese** [comment]
Looks like a regression from #344 

PR to resolve here https://github.com/open-policy-agent/conftest/pull/395

**Comment by Blokje5** [comment]
Fixed by #395
